fabric-recipe-api-v1/src/main/java/net/fabricmc/fabric/impl/recipe/ingredient/SupportedIngredientsPacketEncoder.java

```diff
@@ -26,4 +26,6 @@
  */
 public interface SupportedIngredientsPacketEncoder {
```

Comment:
This interface should be renamed to `SupportedIngredientsClientConnection` to match the name of the class it's actually implemented on.

Reply:
Will change that and also fix the comment as it was still on EncoderHandler